db/crud.go

```diff
@@ -1684,6 +1686,11 @@ func (db *DatabaseContext) assignSequence(ctx context.Context, docSequence uint6
 			if err != nil {
 				return unusedSequences, err
 			}
+			numSkippedSequences := docSequence - doc.Sequence
+			if numSkippedSequences > oversizeSkippedSequenceWarning {
+				base.WarnfCtx(ctx, "doc %s / previous rev: %s /  new rev: %s be allocated sequence %d but standard next sequence allocation would be %d. This is significantly ahead of the previous sequence this document had %d.", base.UD(doc.ID), doc.CurrentRev, newRevID, docSequence, firstAllocatedSequence, doc.Sequence)
```

Comment:
```suggestion
				base.WarnfCtx(ctx, "Doc %s / %s had an existing sequence %d that is higher than the next db sequence value %d, resulting in the release of %d unused sequences. This may indicate documents being migrated between databases by an external process", base.UD(doc.ID), doc.CurrentRev, doc.Sequence, firstAllocatedSequence, unusedSequenceRange)
```